tests/unit/test_arrow.py

```diff
@@ -216,6 +216,7 @@ def types_mapper(
             ],
             type=pyarrow.time64("ns"),
         ),
+        id="time-nanoseconds-arrow-round-trip",
```

Comment:
Leftover from debugging why when I made the "time" `_datetime` method return a `Timestamp` too, this broke. Since that change isn't necessary and it broke this test I reverted it. Kept the test id, since I thought it was useful.

Reply:
Test ID is indeed useful IMO. :+1: